ci/long_running_tests/workloads/many_actor_tasks.py

```diff
@@ -26,7 +26,7 @@
     cluster.add_node(
         redis_port=6379 if i == 0 else None,
         num_redis_shards=num_redis_shards if i == 0 else None,
-        num_cpus=2,
+        num_cpus=10,
```

Comment:
I think 5 should be sufficient right? If so, let's use that so as not to create too many workers.